app/javascript/src/Navigation/components/PF4NavProxy.jsx

```diff
@@ -8,7 +8,12 @@ const Nav = ({id, children}) => (
   </nav>
 )
 
-const NavExpandable = ({id, title, isExpanded, isActive, children}) => {
+const OutdatedIcon = ({title}) => {
+  const isVisibleIcon = title === 'Integration'
+  return isVisibleIcon ? <i className='fa fa-exclamation-triangle' title='The Integration is out of date, promote!'></i> : null
```

Comment:
I'm thinking about using "promote". It will be both staging and production configs that will be out of date, right? Yet, it appears that we currently only use the "promote" for when updating prod.

Reply:
yes, there will be a new button: https://issues.jboss.org/browse/THREESCALE-3539